docker-compose.prod.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  app:
+    image: app-api:latest
+    networks:
+      - caddy_public
+    volumes:
+      - caddy_data:/data
+      - caddy_config:/config
+      - runtime:/app/app/runtime
```

Comment:
But in dev mode it's `- runtime:/app/runtime`, is it ok?

Reply:
Nope. Good catch.